<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guahao</class>
 <widget class="QWidget" name="guahao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>512</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>guahao</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_no">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_id_no">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sex">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_age">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_administrative">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>科室：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_administrative">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>门诊部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>儿科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>妇科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>神经科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>眼科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>耳鼻喉科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>口腔科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>泌尿科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>骨科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>创伤科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内分泌科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>心脏外科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>麻醉科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>皮肤科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>传染病科 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>脑外科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>胸外科</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_tel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_birth">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_birth">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>生日：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_money">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>挂号费：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>诊断日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_date">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_chuzhen">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>初诊</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_fuzhen">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>复诊</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
